tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -330,7 +330,7 @@ tensorflow/lite/micro/kernels/elu.cc \
 tensorflow/lite/micro/kernels/ethosu.cc \
 tensorflow/lite/micro/kernels/exp.cc \
 tensorflow/lite/micro/kernels/fill.cc \
-tensorflow/lite/micro/kernels/flexbuffers_generated_data.cc \
+tensorflow/lite/micro/kernels/detection_postprocess_flexbuffers_generated_data.cc \
```

Comment:
since we're going with explicitly listing the test targets, we should be able to remove this line.

Reply:
Done.